internal/pkg/v2/infrastructure/redis/queries.go

```diff
@@ -18,7 +18,7 @@ import (
 func getObjectById(conn redis.Conn, id string, out interface{}) errors.EdgeX {
 	obj, err := redis.Bytes(conn.Do(GET, id))
 	if err == redis.ErrNil {
-		return errors.NewCommonEdgeX(errors.KindEntityDoesNotExist, fmt.Sprintf("object %T doesn't exist in the database", out), err)
+		return errors.NewCommonEdgeX(errors.KindEntityDoesNotExist, fmt.Sprintf("fail to query object %T, %s doesn't exist in the database", out, id), err)
```

Comment:
("fail to query object %T, because id: %s doesn't exist in the database", out, id)

Reply:
Modified.